cpp/dolfinx/mesh/MeshTags.h

```diff
@@ -75,6 +75,28 @@ class MeshTags
   /// Move assignment
   MeshTags& operator=(MeshTags&& tags) = default;
 
+  /// Find all MeshTags for a list of values and return the corresponding
+  /// indices per value, local-to-process
+  /// @param[in] values A list values
+  /// @return The correspodning indices
+  std::vector<std::vector<std::int32_t>> indices(std::vector<T> values) const
```

Comment:
Add `const` and `&`: `std::vector<std::vector<std::int32_t>> indices(const std::vector<T>& values) const`